site/content/en/docs/tasks/run/pod-based-workloads/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Pod Based Workloads"
```

Comment:
Maybe we could change the title to: "External integrations", or something similar? `Pod Based Workloads` can be a little misleading, since we already have `Plain Pods` section

Reply:
I agree it is worth thinking about a good name, but we already have a concept of "external integrations" / custom jobs: https://kueue.sigs.k8s.io/docs/tasks/dev/integrate_a_custom_job/
So, it could be misleading too.

Reply:
OTOH, now thinking more about this, Plain Pods is one of the mechanisms for supporting external frameworks, as suggested here: https://github.com/kubernetes-sigs/kueue/pull/3897#pullrequestreview-2569382461. 

So, I'm ok with the more generic section like "External integrations", then it should also probably group https://kueue.sigs.k8s.io/docs/tasks/run/flux_miniclusters/ and https://kueue.sigs.k8s.io/docs/tasks/run/wrapped_custom_workload/, and refactor the structure. WDYT @PBundyra @kannon92 ?